src/app/components/shell/router-helpers/fallback-to.js

```diff
@@ -29,16 +23,14 @@ export default function FallbackTo({name}) {
     // i think page-data-utils:fetchFromCMS would have to be updated
     // to do something special on a 404 status
     if ('error' in data) {
-        return <Layout name="default"><Error404 /></Layout>;
+        return <Error404 />;
```

Comment:
the 404 page should probably always be shown in the default layout, should you call `setLayoutParameters` to change it to `default` here?

Reply:
Ok. I also set defaults for `setLayoutParameters` so that without arguments it sets default layout.